erpnext/hr/doctype/job_opening/job_opening.py

```diff
@@ -54,9 +54,10 @@ def get_list_context(context):
 	context.title = _("Jobs")
 	context.introduction = _('Current Job Openings')
 	context.get_list = get_job_openings
+	context.result_heading_template = "bloomstack_website/www/career.html"
```

Comment:
- There should be a better way of overriding Job Opening html, because this might break someday if the website app is not installed.